sdk/lib/3rdparty/freetype/src/type1/t1gload.c

```diff
@@ -312,14 +311,18 @@
     FT_Error       error;
 
 
+    FT_TRACE5(( "T1_Get_Advances:\n" ));
+
     if ( load_flags & FT_LOAD_VERTICAL_LAYOUT )
     {
       for ( nn = 0; nn < count; nn++ )
+      {
         advances[nn] = 0;
 
-#ifdef __REACTOS__
-      free(&decoder);
-#endif
+        FT_TRACE5(( "  idx %d: advance height 0 font units\n",
+                    first + nn ));
+      }
+
       return FT_Err_Ok;
```

Comment:
Please restore the free(&decoder) just before this return.

Reply:
OK.